pandas_ext/snowflake.py

```diff
@@ -0,0 +1,71 @@
+"""Snowflake connection module."""
+from os import getenv
+
+import pandas as pd
+
+from snowflake.sqlalchemy import URL
+from sqlalchemy import create_engine
+
+
+def connect(**kwargs):
+    """Connect to snowflake."""
+    user = getenv('SNOWFLAKE_USER')
+    password = getenv('SNOWFLAKE_PASSWORD')
+    account = getenv('SNOWFLAKE_ACCOUNT')
+    database = getenv('SNOWFLAKE_DATABASE')
+    warehouse = getenv('SNOWFLAKE_WAREHOUSE')
+    role = getenv('SNOWFLAKE_ROLE')
+    return create_engine(
+        URL(
+            account=account,
+            user=user,
+            password=password,
+            database=database,
+            warehouse=warehouse,
+            role=role,
+            **kwargs
+        )
+    )
+
+
+def read_snowflake(sql: str, con=None, **kwargs) -> pd.DataFrame:
+    """Retrieves a query from snowflake.
+    
+    >>> df = read_snowflake('select a from test.test')
+    """
+    if con is None:
+        con = connect()
+    return pd.read_sql(sql, con, **kwargs)
+
+
+def to_snowflake(
+        df,
+        schema,
+        table,
+        con=None,
+        index=False,
+        if_exists='append',
```

Comment:
I think the options for the `if_exists` argument should be documented.

Reply:
They are documented in the `to_sql` function in pandas. I can post the URL to that page.